tests/commands/test_account.py

```diff
@@ -170,15 +170,21 @@ def test_send_sign_transaction_and_execute():
         )
 
 
+def test_send_defaults():
+    account = Account(KEY, NETWORK)
+
+    account.send(account.address, "method", [1, 2, 3])
```

Comment:
Just to be thorough, I'd create a mock for `account.send` and call the mock's `assert_called_once_with` to assert the passed default values.